ui/components/app/hold-to-reveal-button/hold-to-reveal-button.js

```diff
@@ -152,6 +152,7 @@ export default function HoldToRevealButton({ buttonText, onLongPressed }) {
           </svg>
         </div>
         <div
+          aria-label="hold-to-reveal-circle-unlocked"
           className="hold-to-reveal-button__unlock-icon-container"
           onAnimationEnd={triggerOnLongPressed}
         >
```

Comment:
I think a recent update to the Button component broke this component. This should fix it.

```
    <Button
      onMouseDown={onMouseDown}
      onMouseUp={onMouseUp}
      className="hold-to-reveal-button__button-hold"
      textProps={{ display: DISPLAY.FLEX, alignItems: AlignItems.center }}
    >
      <Box className="hold-to-reveal-button__icon-container" marginRight={2}>
        {renderPreCompleteContent()}
        {renderPostCompleteContent()}
      </Box>
      {buttonText}
    </Button>
```

## Before
<img width="832" alt="Screenshot 2023-03-28 at 9 04 38 AM" src="https://user-images.githubusercontent.com/8112138/228301218-14425c0f-fb5f-4778-b5bf-03f0a723c293.png">

## After
<img width="825" alt="Screenshot 2023-03-28 at 9 05 31 AM" src="https://user-images.githubusercontent.com/8112138/228301104-402c9132-adcd-466b-8530-12f040637b69.png">

Reply:
cc @garrettbear I think we might be over complicating the button in this case it uses a custom icon that is passed to the children. The loading and ellipse logic adds a secondary unnecessary wrapper? I think the previous update should have just been changing the root element to a Text component to allow for the text styles. There is now one unnecessary wrapping span here

<img width="832" alt="Screenshot 2023-03-28 at 9 04 38 AM" src="https://user-images.githubusercontent.com/8112138/228301824-33d2194a-50bc-4684-99f4-2d256be017e3.png">